### ISSUE TITLE
Error with self._state = temperature_history.temperatures.pop() in sensor.py

### ISSUE THREAD

**Issue by zaywalker** [syssi/xiaomi_cooker#34]
And unable to track the temperature.

Hello! I'm very happy to use this cooker sensor and I've got a error message from sensor.py.

My cooker is chunmi.cooker.press2.

Here is the error message.
```
Logger: homeassistant.util.logging
Source: util/logging.py:106
First occurred: 오후 4:13:37 (40 occurrences)
Last logged: 오후 4:33:07

Exception in async_update_callback when dispatching 'xiaomi_miio_cooker_updated': ('192.168.10.202',) Traceback (most recent call last): File "/config/custom_components/xiaomi_miio_cooker/sensor.py", line 112, in async_update_callback self._state = temperature_history.temperatures.pop() IndexError: pop from empty list
```

**Comment by tieppm** [comment]
hello
model your xiaomi cooker ??
 Xiaomi YLIH02CM IH 1S 3L chunmi.cooker.k1pro1 ???

**Comment by tieppm** [comment]
hello
model your xiaomi cooker ?
 Xiaomi YLIH02CM IH 1S 3L chunmi.cooker.k1pro1 ?

**Comment by zaywalker** [comment]
> hello
> model your xiaomi cooker ?
> Xiaomi YLIH02CM IH 1S 3L chunmi.cooker.k1pro1 ?

My model is chunmi.cooker.press2. I'm not sure the product code name, I guess my cooker is YLIH01CM which is 1st model from xiaomi(mijia) as a pressure cooker.

**Comment by tieppm** [comment]
oh. YLIH01CM is chunmi.cooker.press1
my model is  YLIH02CM or IH 1S 3L or chunmi.cooker.k1pro1
![image](https://user-images.githubusercontent.com/42457105/111037724-045b1a80-8458-11eb-822d-2d22d561a935.png)
![image](https://user-images.githubusercontent.com/42457105/111037829-7af81800-8458-11eb-9adb-1df0c0bd64fe.png)
![image](https://user-images.githubusercontent.com/42457105/111037879-a844c600-8458-11eb-91df-704588f4bfb8.png)

**Comment by zaywalker** [comment]
Yup. I see. As I mentioned, I'm not sure about the product name. :) Just I know model code that I can check with mi home app.

![H0c4c5f655aeb4e4eab454f7f89cd1d55r](https://user-images.githubusercontent.com/33807772/111038142-b7cd0a80-846a-11eb-9d59-9fbc6a417c59.png)

**Comment by tieppm** [comment]
yes, your is YLIH01CM or chunmi.cooker.press1

**Comment by zaywalker** [comment]
> yes, your is YLIH01CM or chunmi.cooker.press1

I'm sorry. My cooker is chunmi.cooker.press2.

![Screenshot_2021-03-14-02-29-05-251_com xiaomi smarthome](https://user-images.githubusercontent.com/33807772/111038698-c1a43d00-846d-11eb-9807-1386c0bac5cb.jpg)

**Comment by tieppm** [comment]
Oh. You user app? Mi home?

**Comment by zaywalker** [comment]
yup. custom mi home app from "https://www.kapiba.ru/2017/11/mi-home.html" which is region free.

**Comment by tieppm** [comment]
@syssi YLIH01CM is  chunmi.cooker.press2?
but config of you YLIH01CM is  chunmi.cooker.press1
there is something wrong here?
my model is YLIH02CM or IH 1S 3L or chunmi.cooker.k1pro1. it is chunmi.cooker.press1 or chunmi.cooker.press2?
i dont config it vs hass
![111037724-045b1a80-8458-11eb-822d-2d22d561a935](https://user-images.githubusercontent.com/42457105/114481563-0a970f00-9c2f-11eb-843c-399782a7d02f.png)

**Comment by syssi** [comment]
You could setup this component. If your device isn't supported you will see the model name of your device at the homeassistant.log.

**Comment by tieppm** [comment]
> You could setup this component. If your device isn't supported you will see the model name of your device at the homeassistant.log.

thank U, please help me!
![111037969-0b365d00-8459-11eb-8f9c-61cde53f24e8](https://user-images.githubusercontent.com/42457105/114510442-e3593580-9c60-11eb-87eb-1647efe61da9.jpg)
![172264055_1483067015418446_8061271452864163189_n](https://user-images.githubusercontent.com/42457105/114510453-e94f1680-9c60-11eb-85b4-96f9e32ceaad.jpg)


![image](https://user-images.githubusercontent.com/42457105/114509988-59a96800-9c60-11eb-9358-61bd055a0cdb.png)
![image](https://user-images.githubusercontent.com/42457105/114510069-6e85fb80-9c60-11eb-84ea-6717c9db7901.png)
![image](https://user-images.githubusercontent.com/42457105/114510123-7ba2ea80-9c60-11eb-95ef-7f4d371c5858.png)
![image](https://user-images.githubusercontent.com/42457105/114510176-8d848d80-9c60-11eb-9764-8851c1417e30.png)
![111037879-a844c600-8458-11eb-91df-704588f4bfb8](https://user-images.githubusercontent.com/42457105/114510433-e2280880-9c60-11eb-9fb2-734d033674fb.png)

**Comment by tieppm** [comment]
@syssi please help me !!!
